Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM drupal:8.9.2-apache
+FROM drupal:8.9.6-apache AS base
```

Comment:
Not for this PR, but does this mean we can remove the Drupal version requirement from our composer setup? I'm interested in whether you're leaning more towards leaning on the Drupal Docker image or stepping back to the PHP/Apache base image in future iterations?